### DISCUSSION TITLE
HLS Streams and timeshifting

### DISCUSSION THREAD

**Discussion by matthieudraft** [AzuraCast/AzuraCast#6012] [Q&A]
Hello,
I would have liked to know if HLS streams already allowed timeshifting, going back in time, going back to the beginning of a program for example?
Do segments remain cached on the server?
I know that some French radios already offer 4 hour timeshifting on their website.
Regards
Matthieu

**Comment by BusterNeece** [comment]
Right now, we just use Liquidsoap's built-in HLS mechanism, and we configure it to get rid of older chunks rather aggressively, so that audio players won't get hung up on older chunks thinking they're in realtime.

**Reply by matthieudraft** [reply]
Thanks Buster
Can I put this request in the requests for improvement?
